### PR TITLE
Updated Dockerfile for issue#554

### LINKED ISSUE #554: mosquitto.conf.example file not found by iot/mosquitto Dockerfile
Build:
```sudo docker build -t mosquitto -f ./Dockerfile .```

Output:
```
Sending build context to Docker daemon  4.608kB
Step 1/6 : FROM centos:7
7: Pulling from library/centos
74f0853ba93b: Pull complete 
Digest: sha256:26f74cefad82967f97f3eeeef88c1b6262f9b42bc96f2ad61d6f3fdf544759b8
Status: Downloaded newer image for centos:7
 ---> 328edcd84f1b
Step 2/6 : COPY aclfile /root/
 ---> 8a07c5e9d97d
Removing intermediate container 0ad645acf4c5
Step 3/6 : COPY startMosquitto.sh /bin
 ---> c9f37ab86ff4
Removing intermediate container 922f2fc760d0
Step 4/6 : RUN yum update -y && yum install -y wget   && yum install -y epel-release   && yum update -y epel-release   && yum install -y mosquitto   && chmod 755 /bin/startMosquitto.sh   && mkdir /var/log/mosquitto   && chown mosquitto:mosquitto /var/log/mosquitto   && touch /etc/mosquitto/pwfile   && mv /etc/mosquitto/mosquitto.conf.example /etc/mosquitto/mosquitto.conf   && sed -i '$ i acl_file /etc/mosquitto/aclfile\npassword_file /etc/mosquitto/pwfile' /etc/mosquitto/mosquitto.conf   && mv /root/aclfile /etc/mosquitto/aclfile   && yum clean all
 ---> Running in 6c3b9cf08004
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: mirror.airenetworks.es
 * extras: mirror.airenetworks.es
 * updates: mirror.airenetworks.es
No packages marked for update
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: mirror.airenetworks.es
 * extras: mirror.airenetworks.es
 * updates: mirror.airenetworks.es
Resolving Dependencies
--> Running transaction check
---> Package wget.x86_64 0:1.14-13.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package         Arch              Version                Repository       Size
================================================================================
Installing:
 wget            x86_64            1.14-13.el7            base            546 k

Transaction Summary
================================================================================
Install  1 Package

Total download size: 546 k
Installed size: 2.0 M
Downloading packages:
warning: /var/cache/yum/x86_64/7/base/packages/wget-1.14-13.el7.x86_64.rpm: Header V3 RSA/SHA256 Signature, key ID f4a80eb5: NOKEY
Public key for wget-1.14-13.el7.x86_64.rpm is not installed
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
Importing GPG key 0xF4A80EB5:
 Userid     : "CentOS-7 Key (CentOS 7 Official Signing Key) <security@centos.org>"
 Fingerprint: 6341 ab27 53d7 8a78 a7c2 7bb1 24c6 a8a7 f4a8 0eb5
 Package    : centos-release-7-3.1611.el7.centos.x86_64 (@CentOS)
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : wget-1.14-13.el7.x86_64                                      1/1 
install-info: No such file or directory for /usr/share/info/wget.info.gz
  Verifying  : wget-1.14-13.el7.x86_64                                      1/1 

Installed:
  wget.x86_64 0:1.14-13.el7                                                     

Complete!
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: mirror.airenetworks.es
 * extras: mirror.airenetworks.es
 * updates: mirror.airenetworks.es
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:7-9 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                Arch             Version         Repository        Size
================================================================================
Installing:
 epel-release           noarch           7-9             extras            14 k

Transaction Summary
================================================================================
Install  1 Package

Total download size: 14 k
Installed size: 24 k
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : epel-release-7-9.noarch                                      1/1 
  Verifying  : epel-release-7-9.noarch                                      1/1 

Installed:
  epel-release.noarch 0:7-9                                                     

Complete!
Loaded plugins: fastestmirror, ovl
http://mirror.airenetworks.es/epel/7/x86_64/repodata/060cd65ba0a56c60ddaae3f82e8943dacd46aca3ffa3f331c1c03abd91ef2299-updateinfo.xml.bz2: [Errno 14] HTTP Error 404 - Not Found
Trying other mirror.
To address this issue please refer to the below knowledge base article 

https://access.redhat.com/articles/1320623

If above article doesn't help to resolve this issue please create a bug on https://bugs.centos.org/

Loading mirror speeds from cached hostfile
 * base: mirror.airenetworks.es
 * epel: mirror.airenetworks.es
 * extras: mirror.airenetworks.es
 * updates: mirror.airenetworks.es
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:7-9 will be updated
---> Package epel-release.noarch 0:7-10 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                Arch             Version           Repository      Size
================================================================================
Updating:
 epel-release           noarch           7-10              epel            14 k

Transaction Summary
================================================================================
Upgrade  1 Package

Total download size: 14 k
Downloading packages:
Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
warning: /var/cache/yum/x86_64/7/epel/packages/epel-release-7-10.noarch.rpm: Header V3 RSA/SHA256 Signature, key ID 352c64e5: NOKEY
Public key for epel-release-7-10.noarch.rpm is not installed
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
Importing GPG key 0x352C64E5:
 Userid     : "Fedora EPEL (7) <epel@fedoraproject.org>"
 Fingerprint: 91e9 7d7c 4a5e 96f1 7f3e 888f 6a2f aea2 352c 64e5
 Package    : epel-release-7-9.noarch (@extras)
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : epel-release-7-10.noarch                                     1/2 
  Cleanup    : epel-release-7-9.noarch                                      2/2 
  Verifying  : epel-release-7-10.noarch                                     1/2 
  Verifying  : epel-release-7-9.noarch                                      2/2 

Updated:
  epel-release.noarch 0:7-10                                                    

Complete!
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: mirror.airenetworks.es
 * epel: mirror.airenetworks.es
 * extras: mirror.airenetworks.es
 * updates: mirror.airenetworks.es
Resolving Dependencies
--> Running transaction check
---> Package mosquitto.x86_64 0:1.4.13-1.el7 will be installed
--> Processing Dependency: libwebsockets.so.7()(64bit) for package: mosquitto-1.4.13-1.el7.x86_64
--> Processing Dependency: libcares.so.2()(64bit) for package: mosquitto-1.4.13-1.el7.x86_64
--> Running transaction check
---> Package c-ares.x86_64 0:1.10.0-3.el7 will be installed
---> Package libwebsockets.x86_64 0:1.7.5-1.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package               Arch           Version                Repository    Size
================================================================================
Installing:
 mosquitto             x86_64         1.4.13-1.el7           epel         187 k
Installing for dependencies:
 c-ares                x86_64         1.10.0-3.el7           base          78 k
 libwebsockets         x86_64         1.7.5-1.el7            epel         2.5 M

Transaction Summary
================================================================================
Install  1 Package (+2 Dependent packages)

Total download size: 2.7 M
Installed size: 3.4 M
Downloading packages:
--------------------------------------------------------------------------------
Total                                              2.0 MB/s | 2.7 MB  00:01     
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : libwebsockets-1.7.5-1.el7.x86_64                             1/3 
  Installing : c-ares-1.10.0-3.el7.x86_64                                   2/3 
  Installing : mosquitto-1.4.13-1.el7.x86_64                                3/3 
  Verifying  : mosquitto-1.4.13-1.el7.x86_64                                1/3 
  Verifying  : c-ares-1.10.0-3.el7.x86_64                                   2/3 
  Verifying  : libwebsockets-1.7.5-1.el7.x86_64                             3/3 

Installed:
  mosquitto.x86_64 0:1.4.13-1.el7                                               

Dependency Installed:
  c-ares.x86_64 0:1.10.0-3.el7        libwebsockets.x86_64 0:1.7.5-1.el7       

Complete!
mv: cannot stat '/etc/mosquitto/mosquitto.conf.example': No such file or directory
The command '/bin/sh -c yum update -y && yum install -y wget   && yum install -y epel-release   && yum update -y epel-release   && yum install -y mosquitto   && chmod 755 /bin/startMosquitto.sh   && mkdir /var/log/mosquitto   && chown mosquitto:mosquitto /var/log/mosquitto   && touch /etc/mosquitto/pwfile   && mv /etc/mosquitto/mosquitto.conf.example /etc/mosquitto/mosquitto.conf   && sed -i '$ i acl_file /etc/mosquitto/aclfile\npassword_file /etc/mosquitto/pwfile' /etc/mosquitto/mosquitto.conf   && mv /root/aclfile /etc/mosquitto/aclfile   && yum clean all' returned a non-zero code: 1
```

### PR DESCRIPTION
**Pull request by vjain641**
*Edit*: fixes #554 

Removing the mv line as mosquitto.conf file is already present in /etc/mosquitto directory.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,21 +1 @@
...
-    moment: ~2.19.4 -> ~2.22.2
-    mongodb: 2.2.35 -> 3.1.8
-    mongoose: 4.13.12 -> 5.3.6
-    mustache: 2.2.1 -> 3.0.0
-    query-string: 4.3.4 -> 6.2.0
-    request: 2.39.0 - 2.81.0 -> 2.88.0
-    underscore: ~1.7.0 -> ~1.9.1
-    xmldom: 0.1.19 -> 0.1.27
-Remove: old unused dependencies (sax, grunt)
+[iotagent-node-lib] mosquitto.conf.example file not found by iot/mosquitto Dockerfile (#554)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: CHANGEX_NEXT_RELEASE has to be fixed. You shouldn't delete existing lines, just adding the new one. The `[iotagent-node-lib]` prefix is not needed, I'd suggest to use instead:

```
Fix: mosquitto.conf.example file not found by iot/mosquitto Dockerfile (#554)
```

Comment (vjain641) [reply CHANGES_NEXT_RELEASE]: Done Thanks

**Comment by fgalan** [comment]
@dcalvoalonso ups... maybe I merged this too fast @dcalvoalonso, sorry... 

However, any feedback you may have could be included post-merge, of course.

**Comment by fgalan** [comment]
#554 was a bug in an auxiliary dockerfile at docker/Mosquitto/ which is basically a a dockerization of mosquitto MQTT broker. The docker is provide as a help for users to test with MQTT, but it is just an auxiliary material in this repository.

**Comment by dcalvoalonso** [comment]
Maybe I would suggest to include a very very very short README.md file within docker/mosquitto folder explaining this. If not, I am afraid that it is a bit difficult to understand the purpose of this file.

What do you think?

**Pull request merged**